### ISSUE TITLE
How to calculate the final logits and predicted label?

### ISSUE THREAD

**Issue by ZhangVerse** [Chen-GX/CIE#1]
I am very interested in the method proposed in your paper. My current research focuses on fairness, and I am considering applying your approach to fairness in Graph Neural Networks (GNNs). I have some questions regarding the validation or testing phase. Specifically, when computing the final prediction logits, should I use only the o_logits, or is there a need to combine the results of the three components with some weighted sum to obtain the predicted labels?

**Comment by Chen-GX** [comment]
I only use `o_logits` for final prediction in my paper, because it is causal feature.

**Comment by ZhangVerse** [comment]
> I only use `o_logits` for final prediction in my paper, because it is causal feature.

Thanks!
